descriptors/ModuleDescriptor-template.json

```diff
@@ -1318,29 +1318,31 @@
       ]
     },
     {
-      "permissionName" : "orders-storage.claiming.process",
+      "permissionName" : "orders-storage.claiming.process.execute",
       "displayName" : "orders-storage-claiming-process",
-      "description" : "Iterate through claimed pieces and modify their status as needed"
+      "description" : "Iterate through claimed pieces and modify their status as needed",
+      "replaces": ["orders-storage.claiming.process"]
```

Comment:
just for my understanding for which purposes we need "replaces"?

Reply:
for permissions migration, for BE it's not so important as I understand, but nice to have